lib/user.rb

```diff
@@ -0,0 +1,20 @@
+require 'net/http'
+require 'json'
+
+class User
+  attr_accessor :hash, :points
+
+  def initialize(name)
+    @hash = {}
+    uri = URI('https://api.github.com/users/'+ name)
```

Comment:
Surrounding space missing for operator '+'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.